packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -18,14 +18,16 @@ export type AvatarSlots = {
   image?: IntrinsicShorthandProps<'img'>;
 
   /**
-   * The label shown when there's no image. Defaults to the initials derived from `name` using `getInitials`.
+   * (optional) Custom initials. By default, this will be derived from the `name` using `getInitials`.
```

Comment:
```suggestion
   * (optional) Custom initials. By default, this will be derived from the `name`.
```

Reply:
It's still using `getInitials` (the function) but not the prop. I think it's still worth mentioning that function in case someone wants to write their own implementation based on the default `getInitials`.

I did just notice that `getInitials` is not exported. I'll make a follow-up PR for that as well.